src/platform.hpp

```diff
@@ -19,6 +19,7 @@
 #pragma once
 
 #include <string>
+#include <android/asset_manager.h>
```

Comment:
This file is shared across all platforms, so I would be fine with removing this include and just including the declaration in the file where it's used (`src/android/io_realm_react_RealmReactModule.cpp`).